vault.js

```diff
@@ -1,4 +1,54 @@
 'use strict';
 module.exports = function() {
   
-};
\ No newline at end of file
+  let vault = {};
+  const setValue = function(key,value) {
+    if (!key || !value) {
+      return null;
+    } else {
+      vault.key = value;
+    }
+  }
+
+  const getValue = function(key) {
+    if (vault.key) {
+      return vault.key;
+    } else {
+      return null;
+    }
+  }
+
+  return {
+    setValue: setValue,
+    getValue: getValue
+  }
+
+};
+
+
+/*
```

Comment:
When you're done with a particular assignment, it's best to remove all these comments.